ui/prometheus-plugin/src/plugins/prometheus-time-series-query/plugin.test.ts

```diff
@@ -0,0 +1,30 @@
+import { TimeSeriesQueryContext } from '@perses-dev/plugin-system';
+import { PrometheusTimeSeriesQuery } from './';
+
+const stubTimeSeriesContext: TimeSeriesQueryContext = {
+  datasourceStore: {
+    getDatasource: jest.fn(),
+    getDatasourceClient: jest.fn(),
+    listDatasourceMetadata: jest.fn(),
+  },
+  refreshKey: 'test',
+  timeRange: {
+    end: new Date('01-01-2023'),
+    start: new Date('01-02-2023'),
+  },
+  variableState: {},
+};
+
+describe('PrometheusTimeSeriesQuery', () => {
+  it('should properly resolve variable dependencies', () => {
+    if (!PrometheusTimeSeriesQuery.dependsOn) throw new Error('dependsOn is not defined');
+    const { variables } = PrometheusTimeSeriesQuery.dependsOn(
```

Comment:
Trying to write a smoke test but there seems to be an issue with the imports

<img width="915" alt="image" src="https://github.com/perses/perses/assets/6935733/1c9513a6-2afe-4c6b-ab16-d9cbdcd7599d">
@sjcobb any ideas?

Reply:
looks like `jest.mock('echarts/core');` fixed it for now. Need to investigate another way to do this globally for free without the consumer needing to know this